conf/svcomp26/seq.txt

```diff
@@ -0,0 +1,4 @@
+# standard run based on svcomp 25 settings
+svcomp26/level00.json
```

Comment:
I'm wondering if it'd make sense to even make this more general: just a list of Goblint arguments, so `--conf conf/svcomp26/level00.json`.

That would make it easier to reuse confs for Goblint-Par with multishot by writing something like `--conf conf/svcomp26/level00.json --set solver td_something`. Instead of having to make copies of all confs with a single line changed.